docs/lang/articles/misc/gui.md

```diff
@@ -43,8 +45,27 @@ while gui.running:
 
 
 ## Paint on a window
+Taichi's GUI supports painting simple geometric objects, such as lines, triangles, rectangles, circles, and text.
 
-`gui.set_image(pixels)` sets an image to display on the window.
+:::note
+
+The position parameter of every drawing API expects input of 2-element tuples, whose values are the relative position of the object.
+(0.0, 0.0) stands for the lower left corner of the window, and (1.0, 1.0) stands for the upper right corner.
+
+Acceptable input for positions are taichi fields or numpy arrays. Primitive arrays in python are NOT acceptable.
+
+For simplicity, we use numpy arrays in the examples below.
+
+:::
+
+:::tip
+
+For detailed API description, please click on the API code.
```

Comment:
Shall we add an example here to show what is API code? like:

please click on the API code. For instance, click [`gui.get_image()`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=get_image#taichi.misc.gui.GUI.get_image) to see the description to get a GUI image.